torax/sources/source.py

```diff
@@ -216,13 +217,20 @@ def get_value(
         if self.formula is None
         else self.formula
     )
+    # dynamic_source_runtime_params.prescribed_values will be an array
+    # interpolated on the face grid (see config.config_args.interpolate_var_2d)
+    # TODO: This is non-robust, as it assumes the output shape is the cell grid.
```

Comment:
minor nit - just for our own internal linting, this TODO will cause some issues. Can you open up a discussion or issue regarding this, and then we can file it away properly as a TODO?

Here, can you make this a NOTE that this assumes output on a cell grid.

Reply:
Ok! I'll add it as a discussion when the PR's been merged.